### ISSUE TITLE
Slack notifications not firing on AutoPkgr schedule.

### ISSUE THREAD

**Issue by mark-cohen** [lindegroup/autopkgr#515]
With schedules enabled (every 24 hrs), AutoPkgr’s Slack notifications appear to only fire when doing a manual ("Run recipes now") run. Slack test notifications work (immediately).

Hadnt seen any (Slack) notifications (from AutoPkgr) for a while, yet after running AutoPkgr manually, the following notifications were sent/appeared in specified Slack channel:
- "New software available for testing:"
- "Update to core components available:"
- "The following new items were downloaded:"
- 'The following packages were copied:"

Notifications are set to "Report All Items".

Thanks.

**Comment by MichalMMac** [comment]
I had similiar issue with scheduled run notifications after I upgraded to AutoPkgr 1.4. (Manual run worked, test mail worked, scheduled run did not notify at all)
OS reboot "fixed" it.

**Comment by davidtrumbull** [comment]
having the same issue: slack / email options work on manual run but not scheduled runs.

**Comment by homebysix** [comment]
There's a new release candidate for version 1.4.1 in which we think we've squashed a few notification-related bugs. Could you try it out? https://github.com/lindegroup/autopkgr/releases/tag/v1.4.1-rc3

**Comment by davidtrumbull** [comment]
I tried out the rc3, it didn't notify via email or slack, and crashes with a manual "Run Recipes Now". I will reboot later tonight as it is on a production server.

**Comment by homebysix** [comment]
@davidtrumbull It should go without saying, but I'll say it nonetheless: I don't recommend running release candidates on a production server.

**Comment by davidtrumbull** [comment]
thanks hb6. curious thing after I removed the rc3 and went back to release, both slack and email notified.

**Comment by homebysix** [comment]
1.4.1 has been released. If you were experiencing this problem, please upgrade. Thanks!

**Comment by csteelatgburg** [comment]
I upgraded to version 1.4.1 and I am still not receiving notifications in Slack when Autopkgr runs on a schedule. The test message works and when I run manually it works, but I never get anything on the scheduled run. Is there anything else I should check?

**Comment by alexkcal** [comment]
I know this is closed, but I came across this and was still having the same problem with notifications in version 1.4.1 like csteelatgburg. I found another open bug #533 that fixed it for me, so hopefully this helps someone else who may also come across this bug first.

Bug #533 has more details, but the short is that the pref for reportedItemFlags in the plist was not being set. The quick way to set it is to toggle the setting for the actions that will trigger a notification

AutoPkgr Preferences --> Notifications Tab --> Customize button for Choose which actions trigger notification --> Toggle from All items to Some items and then back to All items.

After I did the above, it set reportedItemFlags to 1 and I started receiving the notifications on my scheduled runs.
